Networking/NetworkingTests/Responses/user-complete-without-data.json

```diff
@@ -0,0 +1,13 @@
+{
+  "data": {
```

Comment:
I think we have to remove the "data" envelope.

Reply:
Thanks for noticing! Fixed in 123eacb0892f576abae2621416917ed9d6bd5cd8.